rskj-core/src/main/java/co/rsk/pcc/altBN128/impls/JavaAltBN128.java

```diff
@@ -111,8 +109,7 @@ public int pairing(byte[] data, int length) {
 
         // fail if input len is not a multiple of PAIR_SIZE
         if (data.length % PAIR_SIZE > 0) {
-            output = EMPTY_BYTE_ARRAY;
-            return 1;
+            return returnError();
```

Comment:
The geth impl distinguishes between two different errors here (https://github.com/ethereum/go-ethereum/blob/83e4c49e2b8d3fa83675cf5647e45a6ade7b8b4f/core/vm/contracts.go#L494 )

This is probably useful only for debug. Probably we can have something similar here, but still we need to check that's only a debug thing

Reply:
any of those two errors (!= Revert)[consumes remaining gas](https://github.com/ethereum/go-ethereum/blob/83e4c49e2b8d3fa83675cf5647e45a6ade7b8b4f/core/vm/evm.go#L280), we can add some logs for debug